docs/ConfigureScalarDL.md

```diff
@@ -2,32 +2,39 @@
 
 This guide explains how to configure Scalar DL custom values in helm charts for Scalar DL deployment.
 
-## Create Kubernetes Secrets
+## Update Configuration Files
 
-Kubernetes Secret is an object that contains a small amount of sensitive data such as a password, a token, or a key. 
-This optional method is highly recommended for handling credentials in the production environment.
+### Kubernetes Secrets and Schema Loading reference
 
-You need to create a secret object with `db-username` and `db-password` to store the username and password of the underlying database.
+You can refer to the table below to get values to set create Kubernetes Secrets and update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file properties file according to the database you chose for deployment.
 
-```
-# Cosmos DB
-    # username=<Account name>
-    # password=<Cosmos DB account primary master key>
+| Database  | database | contactPoints              | username          | password                             | dynamoBaseResourceUnit | cosmosBaseResourceUnit | 
+|:----------|----------|----------------------------|-------------------|--------------------------------------|------------------------|------------------------|
+| DynamoDB  | dynamo   | REGION                     | AWS_ACCESS_KEY_ID | AWS_ACCESS_SECRET_KEY                | 10                     | N/A                    |
+| Cosmos DB | cosmos   | Cosmos DB account endpoint | N/A               | Cosmos DB account primary master key | N/A                    | 400                    |
```

Comment:
I don't think it supports only the primary master key.

Reply:
I will add  seconday key along with it.